peachjam/models/core_document_model.py

```diff
@@ -445,19 +445,19 @@ def clean(self):
             .exists()
         ):
             raise ValidationError(
-                {
-                    "expression_frbr_uri": _(
-                        "Document with this Expression FRBR URI already exists!"
-                    )
-                    + " "
-                    + self.expression_frbr_uri
-                }
+                _("Document with this Expression FRBR URI already exists!")
+                + " "
```

Comment:
@longhotsummer I've removed the field level validation. The form mixin does not have the expression_frbr_uri field. Instead of adding it there, removing this field here makes django flash the error at the top of the form, which I think is better since the user cannot edit the expression_frbr_uri field anyway.